src/modules/tasks/task-list/tasks.html

```diff
@@ -8,11 +8,12 @@ <h1 class="bold-text">Tasks</h1>
                     <label class="sr-only" for="filter">Name</label>
                     <div class="input-group">
                         <div class="input-group-btn">
-                            <select ng-init="taskCntrl.filterBy = 'flow'" ng-disabled="taskCntrl.taskList.length === 0" ng-model="taskCntrl.filterBy" class="btn btn-default dropdown-toggle" ng-change="taskCntrl.searchBy = {}">
-                                <option value="flow">Name</option>
+                            <select ng-init="taskCntrl.filterBy = 'job_id'" ng-disabled="taskCntrl.taskList.length === 0" ng-model="taskCntrl.filterBy" class="btn btn-default dropdown-toggle" ng-change="taskCntrl.searchBy = {}">
+                                <option value="job_id">Task ID</option>
+                                <option value="flow">Task</option>
                             </select>
                         </div>
-                        <input type="text" class="form-control" id="filter" ng-model="taskCntrl.searchBy[taskCntrl.filterBy]" placeholder="Filter By {{taskCntrl.filterBy}}...">
+                        <input type="text" class="form-control" id="filter" ng-model="taskCntrl.searchBy[taskCntrl.filterBy]" placeholder="Filter By {{taskCntrl.filterBy === 'job_id'? 'Task Id': '' }}{{taskCntrl.filterBy === 'flow'? 'Task': '' }}...">
```

Comment:
@a2batic Since it has only options to do filtering, you can do  {{taskCntrl.filterBy === 'job_id'? 'Task Id': 'Task' }} instead of adding conditions for both.

Reply:
@gnehapk, Acknowledge